include/enums.h

```diff
@@ -1746,4 +1753,10 @@ enum EffectGfxDataFlags {
     FX_GRAPHICS_2 = 2,
 };
 
+typedef u8 MoveID;
```

Comment:
Thanks for making this enum! I'm not sure we want / should be using the typedefs, because things are inconsistently typed sometimes, and they can make things more complicated than they're worth. Thoughts @nanaian ?

Reply:
Agreed, no need to bother with the typedef. I'm fairly sure we're going to be getting rid of most of them soon anyway